cmd/scollector/conf/conf.go

```diff
@@ -209,6 +210,13 @@ type RedisCounters struct {
 	Database int
 }
 
+type Redis struct {
+	Address      string
+	UseMsec      bool
```

Comment:
This is to optionally prevent pushing floats to opentsdb while retaining current behavior by default. For context, storing float in opentsdb systematically uses 4 bytes [1], integer can go down to a single byte and will only use more if needed. Storing as float provide no additional precision in this case since the full precision of the data is an integer in redis. Considering the time serie using this (all redis calls stats) it will have a mesurable impact on that column size.

1. http://opentsdb.net/docs/build/html/user_guide/backends/hbase.html#data-point-columns